packages/test/test-end-to-end-tests/src/test/summaryDataStoreStats.spec.ts

```diff
@@ -152,15 +160,15 @@ describeNoCompat("Generate Summary Stats", (getTestObjectProvider) => {
         mainDataStore._root.set("dataStore6", dataStore6.handle);
 
         // Wait for summary that contains the above set.
-        let sequenceNumber = await waitForSummary();
+        let sequenceNumber = await waitForSummary(this.timeout());
```

Comment:
You might want to reduce this value since it's called three times in this test.

Reply:
Adjusted the timeout value to fit this case